Readme.asciidoc

```diff
@@ -1,68 +1,116 @@
 
 The services are based on https://www.redhat.com/en/technologies/jboss-middleware/data-grid[JBoss Data Grid] technology and they expose various endpoints and configurations depending on the use case.
 
-What's inside?
---------------
+== Getting started
 
-This repository has the following structure:
+Before you begin working with JDG Online Services, you will need a working OpenShift cluster as well as `oc` client in your `$PATH`. See https://openshift.io/[OpenShift documentation] how to achieve this.
 
-```
-+- Root
-    +-- caching-service - This service
-               +-- image.yaml - Image build recipe
-               +-- Makefile - A set of useful scripts for development work
-               +-- templates - OpenShift templates for this service
-               +-- modules - Modules used for the image
-               +-- functional-tests - Full end-to-end tests with Arquillian Cube
-```
+=== Installing JDG Online Services into Service Catalog
 
-The `image.yaml` uses Python framework called http://concreate.readthedocs.io/en/develop/[Concreate] to build an image. The easiest way to start the build is to invoke `concreate image.yaml target` and invoke a standard Docker build from `target` directory.
+JDG Online Services are supported on OpenShift Container Platform 3.7 (and equivalents such as OpenShift Online). In order to
+ make JDG Online Services available in the Service Catalog, the administrator needs to add all templates into `openshift`
+ namespace. This can be done by invoking `oc create -f templates/caching-service.json -n openshift` and
+ `oc create -f templates/shared-memory-service.json -n openshift` commands.
 
-The `Makefile` contains lots of small, useful scripts. In order to perform a full end-to-end test, invoke `make test-ci`. This will spin up a local OpenShift cluster, build the image, install the template, invoke end-to-end tests and kill the cluster.
 
-The `templates` directory contains a template for Service Catalog. The easiest way to install it is to use `make install-templates-in-openshift-namespace`.
+=== Instantiating JDG Online Service
 
-The `modules` directory contains modules used in `image.yaml`. See http://concreate.readthedocs.io/en/develop/[Concreate] documentation for more details.
+Once the above templates are correctly installed, one can instantiate them using proper icon in the catalog.
 
-The `functional-tests` directory contains http://arquillian.org/arquillian-cube/[Arquillian Cube tests]. The tests assume that there's a running OpenShift cluster and `oc` or `kubectl` is already attached to it.
+image::documentation/img/catalog-walkthrough.gif[JDG Online Service in the Service Catalog]
 
-How can I play with it?
------------------------
+Both Caching and Shared Memory services define a list of optional and required parameters, which have been explained below:
```

Comment:
"The following table describes the parameters for the Caching and Shared Memory services." - I don't think you need to mention that some are required and some are not in that title sentence. It's a bit wordy otherwise. The reader will see that some are required and some are not once they look at the table.

You might consider putting all the required parameters at the top of each table, and all the not required parameters at the bottom. Perhaps an even better option would be have the heading be not "Is required?" but "Required/Optional" and then just put "Required" or "Optional" in the cell/row for each parameter Then it's even more explicit as you read across the row.

Reply:
Done. I added `Required/Optional` header but didn't change the order of parameters. Currently they are sorted the same way as they appear in the template. It's much easier to maintain them this way (I think).